types/marshal.go

```diff
@@ -116,25 +116,9 @@ func (a *Amount) MarshalJSON() ([]byte, error) {
 }
 
 // Sort sorts the response by date, descending
-func (txs TxPage) Len() int           { return len(txs) }
-func (txs TxPage) Less(i, j int) bool { return txs[i].Date > txs[j].Date }
-func (txs TxPage) Swap(i, j int)      { txs[i], txs[j] = txs[j], txs[i] }
-
-// MarshalJSON returns a wrapped list of transactions in JSON
-func (r *TxPage) MarshalJSON() ([]byte, error) {
-	var page struct {
-		Total  int  `json:"total"`
-		Docs   []Tx `json:"docs"`
```

Comment:
just make sure that we return `docs: []` as this in response, instead of `docs: null` when there is no transactions returned. but otherwise it should be lgtm

Reply:
good catch, I added it back